scripts/src/main/resources/scripts/command/node

```diff
@@ -48,24 +69,37 @@ then
 fi
 
 #CLI
-if [ "${1}" = "-h" ] || [ "${1}" = "help" ]
-then
-  echo "Setup or run yarn."
+case ${1} in 
+"help" | "-h")
+  echo "Setup or run NodeJS."
   echo
   echo "Arguments:"
-  echo " setup                    setup angular-cli (install and verify)"
-  echo " get-version              get the current project version"
-  echo " set-version «nv» [«cv»]  set the current project version to new version «nv» (assuming current version is «cv»)"
-  echo " check-top-level-project  check if we are running on a top-level project or fail if in a module or no maven project at all"
-  echo " release                  start a clean deploy release build"
-  echo " «args»                   call maven with the specified arguments"
+  echo " setup                    setup NodeJS (install and verify)"
+  echo " create «name» [«args»]   create a new devon4node application (same as devon4node new)"
+  echo " generate «s» [«args»]    generate devon4node components using the schematic «s» (same as devon4node generate)"
+  echo " db «c» [«args»]          execute a TypeORM command «c» (same as devon4node db)"
+  echo " «args»                   call NodeJS with the specified arguments"
   echo
   echo "Options:"
   exit
-fi
-if [ "${1}" = "setup" ]
-then
+;;
+"setup" | "s")
   doSetup setup
-else
+;;
+"create" | "c")
+  shift
+  doSetupDevon4node setup
+  devon4node new "${@}"
```

Comment:
1. we have 3 such blocks. Therefore it would be nice to have a function to make it only one line (e.g. `doDevon4node`).
2. The `setup` arg IMHO does not seem to make sense here as all calls to `doSetupDevon4Node` are followed by this argument. You can remove it as well as the according if checking for it in `doSetupDevon4Node`.